pom.xml

```diff
@@ -96,6 +96,7 @@
     <commons-io.version>2.7</commons-io.version>
     <license-maven-plugin.version>3.0</license-maven-plugin.version>
     <log4j.version>2.17.1</log4j.version>
+    <log4j.over.slf4j.version>1.7.33</log4j.over.slf4j.version>
```

Comment:
The convention used for other dependencies seems to be `log4j-over-slf4j.version`. Can we use that?

Reply:
Changed.